client/my-sites/stats/overview.jsx

```diff
@@ -82,9 +83,16 @@ class StatsOverview extends Component {
 			);
 		} );
 
+		const date = moment()
+			.utcOffset( 0 ) // Don't have one site to choose a timezone from...
```

Comment:
Perhaps we should default to the account timezone, in that case?

Reply:
I'm not finding an account timezone.
Reviewing this, I'm having difficult understanding why a timezone offset is needed. The stats functions used in the endpoint already apply each particular blog's time zone offset to the time periods it generates. 
This calculation on the client is used to determine which day to ask for. But if I ask for the current day in UTC, the endpoint should take that to mean today, then use the offsets to calculate the correct date windows. I feel like this isn't needed at all, but I could be missing something. I set it to UTC for now (without using moment).

Reply:
Figured out what was happening with the timezone (there was indeed a problem :) ) and came up with a solution. See lower in the main thread for details.